### PR TITLE
Sequencer relay deposits

### PR DESCRIPTION
**Pull request by maxwolff**

### CODE REVIEW

File: params/config.go [hunk 1]
```diff
@@ -256,6 +256,7 @@ var (
 
 	// ScrollAlphaChainConfig contains the chain parameters to run a node on the Scroll Alpha test network.
 	ScrollFeeVaultAddress           = common.HexToAddress("0x5300000000000000000000000000000000000005")
+	ScrollL1MessageQueueAddress     = common.HexToAddress("0x79DB48002Aa861C8cb189cabc21c6B1468BC83BB")
```
Comment (0xmountaintop) [inline params/config.go:259]: is this a fixed address?

File: params/config.go [hunk 2]
```diff
@@ -256,6 +256,7 @@ var (
 
 	// ScrollAlphaChainConfig contains the chain parameters to run a node on the Scroll Alpha test network.
 	ScrollFeeVaultAddress           = common.HexToAddress("0x5300000000000000000000000000000000000005")
```
Comment (0xmountaintop) [inline params/config.go]: we should move this to rollup/rcfg/config.go

**Review by icemelon — COMMENTED** [review]
I realize that we cannot increase the L2 account nonce for L1 messages because the normal L2 txs from the same sender on the L1 will then have an invalid nonce.

File: params/config.go (hunk 1 above)
Comment (icemelon) [reply params/config.go:259]: No, it won't be a fixed address. So we should provide this in the genesis config

File: params/config.go (hunk 2 above)
Comment (Thegaram) [reply params/config.go]: Agreed, we can do it in another PR.

**Pull request closed without merging**